set refresh type on eink update in fillMxcfbUpdateData

houqp opened the pull request.

houqp committed:
set refresh type on eink update in fillMxcfbUpdateData

houqp committed:
add more comments

chrox commented on einkfb.c:
```diff
@@ -97,16 +97,18 @@ inline void fillUpdateAreaT(update_area_t *myarea, FBInfo *fb, lua_State *L) {
 	myarea->update_region.width = luaL_optint(L, 5, fb->vinfo.xres);
 	myarea->update_region.height = luaL_optint(L, 6, fb->vinfo.yres);
 	myarea->waveform_mode = 257;
-	myarea->update_mode = 0;
```
If I guessed right it will do full refresh when `update_mode` equals 0. But it didn't before this fix.

chrox commented:
Anyway I will test this patch right now.

houqp commented:
Oh, wait! seems that it's doing full refresh when update_mode is set to 1. I will test it now...

houqp committed:
adapt refresh type code in fillMxcfbUpdateData

just to keep consistent with the old refresh type, i.e.
0 for full refresh and 1 for partial

houqp commented:
OK, now it should work...

chrox commented:
Thanks. It works now.

chrox commented:
Still don't know how does it work on Kindle Touch.

houqp commented:
Yeah, hopefully it should work :)
